src/actions/actions.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+export const fetchRandomGreeting = () => async (dispatch) => {
+  try {
+    const response = await axios.get('/greetings/random');
```

Comment:
- [ ] Due to the CORS policies I couldn't establish the connection with the API, after fixing the CORS in the API please 🙏🏽 ensure that the front-end works as expected.